arangodb-net-standard/Serialization/JsonNetApiClientSerialization.cs

```diff
@@ -56,6 +58,18 @@ public override byte[] Serialize<T>(T item, ApiClientSerializationOptions serial
                 NullValueHandling = serializationOptions.IgnoreNullValues ? NullValueHandling.Ignore : NullValueHandling.Include
             };
 
+            if (serializationOptions.UseStringEnumConversion)
+            {
+                var stringEnumConverter = new StringEnumConverter();
+                // We assume the UseCamelCasePropertyNames option
+                // should also be applied to string-serialized enums
+                if (serializationOptions.UseCamelCasePropertyNames)
```

Comment:
Is this required because the contract resolver below doesn't apply for converters? I would have thought that `CamelCasePropertyNamesExceptDictionaryContractResolver` would handle that.

Reply:
I had assumed it was necessary but having just checked, turns out you are right and it isn't needed.  Tests pass with or without this code. So I've removed it.